code/modules/holiday/holidays.dm

```diff
@@ -48,9 +61,6 @@
 
 // Return 1 if this holidy should be celebrated today
 /datum/holiday/proc/shouldCelebrate(dd, mm, yyyy, ddd)
-	if(always_celebrate)
```

Comment:
Why did we remove this?

Reply:
As the comment said, it's a debug flag if you want to test christmas. I don't want to need to reimplement it any time I need to test a holiday.